samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -7,28 +7,28 @@ An example pipeline with only [train component](https://github.com/kubeflow/pipe
 2. Get and store data in S3 buckets. You can get sample data using this code.  
    Create a new file `s3_sample_data_creator.py` with following content :
    ```buildoutcfg
+   import io
+   import boto3
    import pickle, gzip, numpy, urllib.request, json
    from urllib.parse import urlparse
-
-   # Load the dataset
-   urllib.request.urlretrieve("http://deeplearning.net/data/mnist/mnist.pkl.gz", "mnist.pkl.gz")
-   with gzip.open('mnist.pkl.gz', 'rb') as f:
-       train_set, valid_set, test_set = pickle.load(f, encoding='latin1')
-
-
-   # Upload dataset to S3
    from sagemaker.amazon.common import write_numpy_to_dense_tensor
-   import io
-   import boto3
 
+   
    ###########################################################################################
    # This is the only thing that you need to change in this code 
    # Give the name of your S3 bucket 
    # To use the example input below give a bucket name which is in us-east-1 region 
    bucket = '<bucket-name>' 
 
    ###########################################################################################
-   
+      
+   # Load the dataset
+   urllib.request.urlretrieve("http://deeplearning.net/data/mnist/mnist.pkl.gz", "mnist.pkl.gz")
+   with gzip.open('mnist.pkl.gz', 'rb') as f:
+       train_set, valid_set, test_set = pickle.load(f, encoding='latin1')
```

Comment:
Why did this move?

Reply:
it's better to have the thing that they need to change at the top rather than in the middle